Code/C++/pangram.cpp

```diff
@@ -65,7 +65,10 @@ int main()
     
     Output: Yes,the string is a pangram
 */
-
+/*
```

Comment:
do not use multiline comments at every step. Use only one opening and closing comment.

Reply:
@rudrakshi99  is it ok now?